object-models/common-models.md

```diff
@@ -17,6 +17,7 @@ KEY | TYPE | NULL/EMPTY | DESCRIPTION
 --- | --- | --- | ---
 code | string | no | Code name for the city. Example: `NEW_YORK`, `DUBAI`
 name | string | no | Display name of the city
+image | [`image`](common-models.md#image) | no | The primary image for the city.
```

Comment:
I think we created the image object for scenarios when we need to represent the image with other properties too. Like caption etc. Question: Should we use just `imageUrl` over here or the object?

Reply:
1. we didn't have that requirement till now.
2. an object will be forward compatible and image url wont, works if new attributes need to be added or not.

Reply:
Let's also add `image` field in Response example of CityAPI [here](https://github.com/headout/api-docs/blob/master/apis/city.md)

Reply:
Let's also add `image` field in Example response of CityAPI [here](https://github.com/headout/api-docs/blob/master/apis/city.md)